src/components/OCNotificationMessage.vue

```diff
@@ -68,17 +53,33 @@ export default {
       required: false,
       default: null,
     },
+    /**
+     * Number of seconds the message shows. It will disappear after this time.
+     */
+    timeout: {
+      type: Number,
+      required: false,
+      default: 5,
```

Comment:
Last tiny bit: please add a validation that the timeout has to be greater than 0. Otherwise it would be possible to create notifications that won't disappear, ever.